docs/instrument.md

```diff
@@ -0,0 +1,49 @@
+
+## Producing instrumented source
+
+The `nyc instrument` command can produce instrumented source files.
+These files are suitable for client side deployment during end to end testing.
+You can either pre-instrument your source, or write instrumented source to a stream.
+
+Run `nyc instrument --help` to display a full list of available command options.
+ 
+### Pre-instrumented source 
+ 
+You can create pre-instrumented source code by running:
+
+```bash
+nyc instrument <input> [output]
+```
+
+`<input>` can be any file or directory within the project root directory.
+The `[output]` directory is optional and can be located anywhere, if not set the instrumented code will be sent to `stdout`.
+For example, `nyc instrument . ./output` will produce instrumented versions of any source files it finds in `.` and store them in `./output`.
+
+The `--delete` option will remove existing output.
+
+The `--complete-copy` option will copy all files from the `input` directory to the `output` directory.
```

Comment:
Will it override the instrumented output? I think more specificity would be good here.

Reply:
That seems obvious to me, I don't see why you'd run `nyc instrument` to simply copy files and not instrument them.  But it can change if it needs to.

Reply:
That's what I mean, I don't really understand from the wording... I assume the output will be `input.js` and `input-instrumented.js` but I don't know if that's actually right